xgcm/test/test_faceconnections.py

```diff
@@ -7,11 +7,14 @@
 
 @pytest.fixture(scope="module")
 def ds():
-    N = 25
+    N = 25  # TODO: Reduce the size here. I think something like 4 or 5 is sufficient
```

Comment:
God yeah it doesn't need to be anywhere near that big. I think 5 at most, at least if you want to test moving two boundaries, each of which could be flipped (so need to be length >=2).

Reply:
Definitely do test changes in a later PR